MCprep_addon/conf.py

```diff
@@ -210,6 +219,183 @@ def deprecation_warning(self):
 
 env = MCprepEnv()
 
+
+# ! Deprecated as of MCprep 3.5
+def init():
```

Comment:
Let's check diffs/make sure dev is merged in - I'll maybe hold off on further review/comments below until that's done